### DISCUSSION TITLE
epoptes on other S.O.

### DISCUSSION THREAD

**Discussion by anescdev** [epoptes/epoptes#139] [Q&A]
How can i install epoptes on Fedora? I try that rpm: https://pkgs.org/search/?q=epoptes.
But i receive that.
![image](https://user-images.githubusercontent.com/74417326/133519973-bb01e23c-814d-47ee-a48b-f8e23f674968.png)

**Comment by alkisg** [comment]
Hi, see https://github.com/epoptes/epoptes/issues/106.
